File: php/test/ExistsTest.php
<?php
declare(strict_types=1);

// TranslateFunnyLanguages SDK exists test

require_once __DIR__ . '/../translatefunnylanguages_sdk.php';

use PHPUnit\Framework\TestCase;

class ExistsTest extends TestCase
{
    public function test_create_test_sdk(): void
    {
        $testsdk = TranslateFunnyLanguagesSDK::test(null, null);
        $this->assertNotNull($testsdk);
    }
}

File: lua/test/exists_test.lua
-- TranslateFunnyLanguages SDK exists test

local sdk = require("translate-funny-languages_sdk")

describe("TranslateFunnyLanguagesSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
